portal/v2/src/Kubeconfig.tsx

```diff
@@ -29,7 +29,7 @@ export const KubeconfigButton = forwardRef<any, KubeconfigButtonProps>(
         if (result?.status === 200) {
           const blob = new Blob([result.request.response])
           const fileDownloadUrl = URL.createObjectURL(blob)
-          const filename = parseContentDisposition(result.headers["content-disposition"]).parameters
+          const filename = parseContentDisposition(result.headers["content-disposition"] || '').parameters
```

Comment:
Had to be added due to an updated package throwing an error on build. On the off chance that undefined is passed to parseContentDisposition, it wouldn't be happy because it requires a string. I have regenerated the error for verification below:

```
TS2345: Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
    Type 'undefined' is not assignable to type 'string'.
        > 32 |           const filename = parseContentDisposition(result.headers["content-disposition"]).parameters
```

Reply:
Ah no worries, all good then